test/e2e/cypress/integration/single-backend/users.spec.js

```diff
@@ -108,12 +108,11 @@ describe('Users', function() {
       .type('{selectall}{backspace}', { delay: 200, force: true })
       .type(
         `{
-"query": { 
+"query": {
```

Comment:
Wrong indentation

Reply:
Indentation here is not added since the ACE editor automatically adds it while Cypress types the text inside it.